test/source/tests/settings.ts

```diff
@@ -425,6 +460,29 @@ export let defineSettingsTests = (testVariant: TestVariant, testWithBrowser: Tes
       await settingsPage1.close();
     }));
 
+    ava.default('settings - email change', testWithBrowser('ci.tests.gmail', async (t, browser) => {
+      const acct1 = 'ci.tests.gmail@flowcrypt.test';
+      const acct2 = 'user@forbid-storing-passphrase-org-rule.flowcrypt.test';
+      const settingsPage = await browser.newPage(t, TestUrls.extensionSettings(acct1));
+      await SettingsPageRecipe.toggleScreen(settingsPage, 'additional');
+      const experimentalFrame = await SettingsPageRecipe.awaitNewPageFrame(settingsPage, '@action-open-module-experimental', ['experimental.htm']);
+      const { cryptup_citestsgmailflowcrypttest_passphrase_07481C8ACF9D49FE: savedPassphrase1 } =
+        await settingsPage.getFromLocalStorage(['cryptup_citestsgmailflowcrypttest_passphrase_07481C8ACF9D49FE']);
+      expect(savedPassphrase1).not.to.be.an('undefined');
+      await experimentalFrame.waitAndClick('@action-change-email');
+      const oauthPopup = await browser.newPageTriggeredBy(t, () => PageRecipe.waitForModalAndRespond(experimentalFrame, 'confirm',
+        { contentToCheck: 'email address has changed', clickOn: 'confirm' }));
+      await OauthPageRecipe.mock(t, oauthPopup, acct2, 'override_acct');
+      await PageRecipe.waitForModalAndRespond(experimentalFrame, 'confirm',
+        { contentToCheck: 'email from ci.tests.gmail@flowcrypt.test to user@forbid-storing-passphrase-org-rule.flowcrypt.test', clickOn: 'confirm' });
+      await PageRecipe.waitForModalAndRespond(experimentalFrame, 'info',
+        { contentToCheck: 'Email address changed to user@forbid-storing-passphrase-org-rule.flowcrypt.test', clickOn: 'confirm' });
+      const { cryptup_userforbidstoringpassphraseorgruleflowcrypttest_passphrase_07481C8ACF9D49FE: savedPassphrase2 } =
+        await settingsPage.getFromLocalStorage(['cryptup_userforbidstoringpassphraseorgruleflowcrypttest_passphrase_07481C8ACF9D49FE']);
+      expect(savedPassphrase2).to.be.an('undefined');
```

Comment:
This is a failing test I added. Should we prevent migration of `cryptup_citestsgmailflowcrypttest_passphrase_07481C8ACF9D49FE` to `cryptup_userforbidstoringpassphraseorgruleflowcrypttest_passphrase_07481C8ACF9D49FE`
by
```
    await AcctStore.set(newAcctEmail, oldAcctStorage);
```

Reply:
Maybe we can take advantage of this: add a comment

```ts
await AcctStore.set(newAcctEmail, oldAcctStorage); // also copies stored pass phrases
```

Then remove all passphrase-handling code in the migration function, except at the end add:

```ts
if(... org rules forbid storing ...) {
  // remove all pass phrases from new account's storage
}
```

Reply:
That would simplify the code quite a bit